libs/@guardian/core-web-vitals/.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+	"extends": ["../../../.eslintrc.js"],
+	"ignorePatterns": ["!**/*", "node_modules"],
+	"overrides": [
+		{
+			"files": ["*.ts", "*.tsx"],
+			"parserOptions": {
+				"project": ["libs/@guardian/core-web-vitals/tsconfig.json"]
+			},
+			"rules": {}
+		},
+		{
+			"files": ["*.ts", "*.tsx"],
+			"rules": {}
+		},
+		{
+			"files": ["*.js", "*.jsx"],
+			"rules": {}
+		},
```

Comment:
What do these overrides do?

Reply:
nothing here, they're duplicates of the other configs though - they mainly provide ways in for people in the future. theyc oudl go though